src/lib.rs

```diff
@@ -1,26 +1,23 @@
-#![feature(conservative_impl_trait, universal_impl_trait)]
-//#![feature(trait_alias)]
```

Comment:
this can be removed  I suppose?

Reply:
I think this is an intermediate diff and is removed by the final step. I do not see "trait_alias" anywhere in the final form of the code.

BTW.. the change at 0e475ed resolves https://github.com/MrBuddyCasino/mcp9808-rs/issues/1